spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/repository/it/ParallelSpannerRepositoryIntegrationTests.java

```diff
@@ -66,7 +66,7 @@ public void testParallelOperations() {
 				assertThat(repo.count()).isZero();
 			});
 			assertThat(repo.count()).isZero();
-			return null;
+			return 0;
```

Comment:
What were the errors that we are fixing?

Reply:
```
[ERROR] testParallelOperations  Time elapsed: 0.243 s  <<< ERROR!
org.springframework.dao.EmptyResultDataAccessException: Result must not be null!
	at org.springframework.cloud.gcp.data.spanner.repository.it.ParallelSpannerRepositoryIntegrationTests.testParallelOperations(ParallelSpannerRepositoryIntegrationTests.java:59)
```

The test entity is shared among several tests, so the `@NotNull` did its job too well.